merlin/models/tf/inputs/embedding.py

```diff
@@ -347,6 +347,10 @@ def Embeddings(
     infer_embedding_sizes: bool = True,
     infer_embedding_sizes_multiplier: float = 2.0,
     infer_embeddings_ensure_dim_multiple_of_8: bool = True,
+    non_trainable: List[str] = [],
```

Comment:
Wondering if it would be more familiar to be called `trainable` to be consistent with the layers?

Passing as a `Dict[str, bool]`. For example `trainable = {"col_a": False}`

Reply:
makes sense.